docs/standard/assembly/index.md

```diff
@@ -0,0 +1,55 @@
+  
+ Assemblies have the following properties:  
+  
+-   Assemblies are implemented as .exe or .dll files.  
+  
+-   For libraries that target the .NET Framework, you can share an assembly between applications by putting it in the global assembly cache. Assemblies must be strong-named before they can be included in the global assembly cache. For more information, see [Strong-Named Assemblies](../../framework/app-domains/strong-named-assemblies.md).  
+  
+-   Assemblies are only loaded into memory if they are required. If they are not used, they are not loaded. This means that assemblies can be an efficient way to manage resources in larger projects.  
+  
+-   You can programmatically obtain information about an assembly by using reflection. For more information, see [Reflection (C#)](../../csharp/programming-guide/concepts/reflection.md) or [Reflection (Visual Basic)](../../visual-basic/programming-guide/concepts/reflection.md).   
+  
+-   You can load an assembly only to inspect it by calling a method <xref:System.Reflection.Assembly.ReflectionOnlyLoadFrom%2A?displayProperty=nameWithType>.  
+  
+## Assembly manifest  
+ Within every assembly is an *assembly manifest*. Similar to a table of contents, the assembly manifest contains the following:  
+  
+-   The assembly's identity (its name and version).  
+  
+-   A file table describing all the other files that make up the assembly, such as another assemblies you created that your .exe or .dll file relies on, or even bitmap or Readme files.  
+  
+-   An *assembly reference list*, which is a list of all external dependencies — .dlls or other files your application needs that may have been created by someone else. Assembly references contain references to both global and private objects. Global objects are available to all other applications. In .NET Core, they are coupled with a particular .NET Core runtime. In .NET Framework, they reside in the global assembly cache. The <xref:Microsoft.VisualBasic?displayProperty=nameWithType> namespace is an example of an assembly in the global assembly cache. Private objects must be in a directory at either the same level as or below the directory in which your application is installed.  
+  
+ Because assemblies contain information about content, versioning, and dependencies, they reduce the applications you create with C# or Visual Basic do not rely on Windows registry values to function properly. Assemblies reduce .dll conflicts and make your applications more reliable and easier to deploy. In many cases, you can install a .NET-based application simply by copying its files to the target computer. For more information, see [Assembly Manifest](../../framework/app-domains/assembly-manifest.md).  
```

Comment:
```suggestion
 Because assemblies contain information about content, versioning, and dependencies, the applications that use them need not rely don Windows registry values to function properly. Assemblies reduce .dll conflicts and make your applications more reliable and easier to deploy. In many cases, you can install a .NET-based application simply by copying its files to the target computer. For more information, see [Assembly Manifest](../../framework/app-domains/assembly-manifest.md).  
```